registries/adapters/openshift_adapter_test.go

```diff
@@ -29,6 +29,7 @@ import (
 
 var Images = []string{"satoshi/nakamoto", "foo/bar", "paul/atreides"}
 var TestURL = "https://registry.reg-aws.openshift.com:443"
+var TestHttpURL = "http://registry.reg-aws.openshift.com:443"
 var RegistryName = "registry.reg-aws.openshift.com:443"
```

Comment:
remove TestURL, TestHttpURL, and RegistryName, just define them in the testcase directly.